package.json

```diff
@@ -27,19 +21,28 @@
     "turbo": "^1.12.5",
     "typescript": "^5.3.0"
   },
+  "engines": {
+    "node": "v20.11.1"
+  },
+  "packageManager": "yarn@4.2.2",
   "resolutions": {
-    "@react-native-community/cli": "^12.3.0",
-    "@react-native-community/cli-platform-android": "^12.3.0",
-    "@react-native-community/cli-platform-ios": "^12.3.0",
-    "@react-native/babel-preset": "^0.73.0",
-    "@react-native/community-cli-plugin": "^0.73.0",
-    "@react-native/community-cli-plugin/@react-native-community/cli-server-api": "^12.3.0",
-    "@react-native/community-cli-plugin/@react-native-community/cli-tools": "^12.3.0",
-    "body-parser/qs": "^6.7.3",
-    "find-babel-config/json5": "^2.1.1",
-    "uuid": "^9.0.0",
-    "ts-node": "^10.9.1",
-    "recursive-readdir/minimatch": "^3.0.5",
-    "serve-handler/minimatch": "^3.0.5"
-  }
+    "@react-native-community/cli": "^14.0.0",
+    "@react-native-community/cli-platform-android": "^14.0.0",
+    "@react-native-community/cli-platform-ios": "^14.0.0",
+    "@react-native/assets-registry": "^0.75.0",
+    "@react-native/babel-plugin-codegen": "^0.75.0",
+    "@react-native/babel-preset": "^0.75.0",
+    "@react-native/codegen": "^0.75.0",
+    "@react-native/community-cli-plugin": "^0.75.0",
+    "@react-native/community-cli-plugin/@react-native-community/cli-server-api": "^14.0.0",
+    "@react-native/community-cli-plugin/@react-native-community/cli-tools": "^14.0.0",
+    "@react-native/gradle-plugin": "^0.75.0",
+    "@react-native/js-polyfills": "^0.75.0",
+    "@react-native/normalize-colors": "^0.75.0",
+    "@react-native/virtualized-lists": "^0.75.0",
+    "find-babel-config/json5": "^2.1.1"
+  },
+  "workspaces": [
+    "packages/*"
```

Comment:
actually, can we explicitly link packages here? I got bitten by this recently 🙈

Reply:
like, `packages/default-storage, packages/sqlite-storage, packages/website, packages/eslint-config`